sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java

```diff
@@ -63,4 +63,10 @@ public void logoutTest() {
         Assert.assertEquals(username, id);
     }
 
+    public String IncrementalConsent(String endpoint) {
```

Comment:
How about `httpGetWithIncreamentalConsent`.
1. First character use lower case.
2. `httpGet` explains what this method did.